### DISCUSSION TITLE
Some related databases, tools/packages and e-books.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [goodluck1982/SpaceGroupIrep#25] [General]
Here are some related databases, tools/packages and e-books, which have the same or overlapping application fields as the `SpaceGroupIrep` package:

- Group theory tools/packages:
https://gtpack.org/
https://gitlab.kwant-project.org/qt/qsymm
https://github.com/nathancarter/group-explorer
https://github.com/gap-system/gap

- Atomly 材料科学数据库 (用高通量第一性原理，推动材料科学演进):
https://atomly.net/#/

- Topological Materials Databases:
https://www.materialscloud.org/discover/topomat
http://materiae.iphy.ac.cn/
http://tm.iphy.ac.cn/opensource.html
https://topgen.nju.edu.cn/
https://extopo.com/materials
https://www.topologicalquantumchemistry.com/#/
https://www.topologicalquantumchemistry.fr/magnetic/

- 常见材料数据库网站 (Please also note that there are some additions in the follow-up on the page below):
https://blog.sciencenet.cn/blog-1502061-1179947.html

- Several books on group theory, condensed matter, fundamental physics, etc.
https://github.com/hongyi-zhao/ebooks

**Comment by goodluck1982** [comment]
Thank you for your information. It's a good collection for reference.
